app/goprotoc/codegen.go

```diff
@@ -257,7 +399,7 @@ type insertedContent struct {
 	lang string
 }
 
-func applyInsertions(fileName string, contents io.Reader, insertions map[string][]insertedContent) (io.Reader, error) {
```

Comment:
nit: you call this `fileName` elsewhere here, maybe keep it the same here and stay with `fileName` instead of `file`?

Reply:
Fair enough. In the other place, the `fileName` was simply relative to the specified output dir, whereas this is now an absolute path that could even refer to a file inside of a zip/jar archive.